editor/src/core/model/performance-scripts.tsx

```diff
@@ -623,11 +623,16 @@ export function useTriggerAbsoluteMovePerformanceTest(
             bubbles: true,
             cancelable: true,
             metaKey: false,
-            clientX: targetBounds.left + (20 + moveCount * 3),
-            clientY: targetBounds.top + (20 + moveCount * 4),
+            clientX: targetBounds.left + (20 + 10 + moveCount * 3),
+            clientY: targetBounds.top + (20 + 10 + moveCount * 4),
             buttons: 1,
           }),
         )
+        const newBounds = targetElement.getBoundingClientRect()
+        if (newBounds.left !== targetBounds.left + 10 + moveCount * 3) {
+          console.info('ABSOLUTE_MOVE_TEST_ERROR')
```

Comment:
This should probably be either a `console.error` or a `throw new Error`

Reply:
I believe the test runner knows what to do with this comment, I copy-pasted it from other error handling code a few lines above